src/other/index.jl

```diff
@@ -313,15 +313,20 @@ function _norm_eq(a1::String, a2::String, idx::Symbol)
             refc1 = get(_julia_charmap, c1, c1)
             refc2 = get(_julia_charmap, c2, c2)
             @assert refc1 == refc2
+            if VERSION >= v"1.8-"
+                suffix = "use the Unicode.normalize function setting its " *
+                         "chartransform keyword argument to Unicode.julia_chartransform."
```

Comment:
How about a more direct suggestion like this?
```suggestion
                suffix = "normalize column names of the data frame by calling " *
                         "`using Unicode; rename!(n -> Unicode.normalize(n, chartransform=Unicode.julia_chartransform), df)`."
```

Reply:
This is a half-measure solution (just like the one below earlier). I will propose a better error (as we need to signal the user if the passed name or the name in data frame was incorrect)

Reply:
@nalimilan In the end I have reverted the simpler error message. The issue is that the number of potential cases we would have to handle is too large to warrant covering all of them given how rare the case is. I now just write which character should be replaced by which.

Just to give you a flavor of what happens:

`"\u0387"` after standard normalization via `Unicode.normalize` becomes `"\u00B7"` which after Julia-specific normalization (`chartransform=Unicode.julia_chartransform`) becomes "\u22C5".

I think that the problematic case is rare enough that we can just write what characters do not match and let the user decide how to handle this.

Reply:
OK, I've lost track of all the possible cases to handle...